### DISCUSSION TITLE
Casa OS setup errors.

### DISCUSSION THREAD

**Discussion by mynted** [tteck/Proxmox#2101] [Help Needed] [answered]
Hi, I'm getting the following errors when attempting to setup Casa OS:

[ERROR] in line 24: exit code 1: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Cheers,

**Comment by tteck** [comment]
When encountering the error message `[ERROR] in line 24: exit code *: while executing command "$@" > /dev/null 2>&1`, make sure to run the script in verbose mode to accurately determine the underlying issue.

**Comment by praveencbe0** [comment]
Faced with the same issue. Ran multiple times in verbose.

Job for casaos-local-storage.service failed because the control process exited with error code. See "systemctl status casaos-local-storage.service" and "journalctl -xeu casaos-local-storage.service" for details.

Running in proxmox 8.1 with default settings for casaos install

**Reply by tteck** [reply]
What is the underlying file system you are using?

**Reply by praveencbe0** [reply]
Internal- local - zfs raid1 (nvme ssd)

**Reply by tteck** [reply]
It appears the issue is in their (`curl -fsSL https://get.casaos.io | sudo bash`) install script. [Others are having the same issue](https://github.com/IceWhaleTech/CasaOS/issues/1461) I'll investigate.

**Reply by tteck** [reply]
~Temporarily removed from the website.~

**Reply by praveencbe0** [reply]
Yes thats the same issue we are facing.

**Reply by tteck** [reply, marked as answer]
Fixed
pinned install script to v0.4.1, you can then update successfully within CasaOS

**Comment by darius97** [comment]
Hello! So im facing the same issue, tried use verbose mode with debian, still fail since i think there is a problem with the debian repo, it created the container but still no casa os installed, tried it several times with many variety but keep using debian... 

then i tried accesing the casaos container and copy the script manually, its has progress but still keep failing installing the casaos itself, but i run the casaos install command from thescript multiple time, it works but when its finished installing, it doesnt show the IP to access it... so i just delete the container..

**But Something did works.** 
So I just use the Advance mode, **Choose Ubuntu**, and choose version 20.04 something , and choose verbose. Its install seamlessly wtihout failing. and gave the ip directly.  and i have been using it and has no problem

**Reply by darius97** [reply]
kudos for the script btw!
